apps/web-staking/src/app/components/ui/buttons/PrimaryButton.tsx

```diff
@@ -23,7 +24,8 @@ export const PrimaryButton = ({
                                 wrapperClassName,
                                 colorStyle,
                                 spinner,
-                                icon
+                                icon,
+                                rightIcon,
```

Comment:
We should not be updating the ui components in the web staking app, instead we need to import them from the ui library